docs/data/material/customization/palette/palette.md

```diff
@@ -108,16 +108,41 @@ const theme = createTheme({
 });
 ```
 
-As in the example above, if the palette color contains custom colors using any of the
-"main", "light", "dark" or "contrastText" keys, these map as follows:
+For the default palette colors (the ones listed in the [Default values section](#default-values) above), only the `main` token is required. If the other tokens are not provided their value is calculated as follows:
```

Comment:
Give each sentence a new line to make line-by-line reviews simpler (older docs don't always adhere to this practice, but it's worth editing whenever you see it). The Markdown only inserts an actual line break after two new lines.

Also, it's always a good idea to explicitly state "if...then" for the benefit of folks who don't have strong English language skills—the sentence's clauses can be difficult to parse otherwise.

```suggestion
For the default palette colors (the ones listed in the [Default values section](#default-values) above), only the `main` token is required. 
If the other tokens aren't provided, then their values are calculated as follows:
```